cmd/dep/init.go

```diff
@@ -258,6 +262,24 @@ func getProjectData(ctx *dep.Ctx, pkgT gps.PackageTree, cpr string, sm *gps.Sour
 				return projectData{}, errors.Wrap(err, "sm.DeduceProjectRoot") // TODO: Skip and report ?
 			}
 
+			start := time.Now()
+			message := make(chan string, len(v.P.Imports))
+			go func() {
+				goStart := time.Now()
+				defer fmt.Fprintf(os.Stderr, "Go Duration: %s\n", time.Since(goStart))
+				err2 := sm.SyncSourceFor(gps.ProjectIdentifier{ProjectRoot: pr})
```

Comment:
As noted in slack, I probably need to add an sm method that more accurately reports the existence of the repo - does it exist upstream? what about on disk?